ansible_collections/arista/avd/molecule/eos_designs_unit_tests/intended/configs/cv-pathfinder-transit.cfg

```diff
@@ -19,7 +19,32 @@ hostname cv-pathfinder-transit
 !
 router path-selection
    !
+   path-group INET id 101
+      ipsec profile CP-PROFILE
+      !
+      local interface Ethernet1
+         stun server-profile cv-pathfinder-pathfinder-INET-0
+      !
+      peer dynamic
+      !
+      peer static router-ip 192.168.44.1
+         name cv-pathfinder-pathfinder
+         ipv4 address 10.7.7.7
+   !
+   path-group MPLS id 100
```

Comment:
The ipsec cli is a bit complicated. You can enabled encryption for static and dynamic tunnels separately (control plane vs data plane), albeit in slightly a complicated manner - 

ipsec enabled under pathgroup enables ipsec for all static and dynamic tunnels. IPSec can then be disabled for all dynamic tunnels by specifying ipsec disabled under peer dynamic. It isn't clear what ipsec:true/false for a path group actually represents. It's possible that the the static tunnel from edge to pathfinder are ipsec encrypted (for a site with MPLS only links that has to connect to a pathfinder with internet links, the actual path would involve going via some gateway to reach the pathfinder through internet and people would want such tunnels to be ipsec encrypted)., but the dynamic tunnels are not. 

How would be enable that? An option would be to always enable ipsec encryption for tunnels from edge to pathfinder and then the existing ipsec attribute only affects the dyanmic tunnels (whether we generate the ipsec disabled cli or not). This is what we did initially, but this has the con that any static paths you create to interconnect the pathfinders themselves using whatever pathgroups it has, they'll have to be ipsec encrypted.

Reply:
tracked as future work